upgrade-tidb-using-tiup.md

```diff
@@ -12,6 +12,13 @@ This document is targeted for the following upgrade paths:
 - Upgrade from TiDB 5.1 versions to TiDB 5.3 versions.
 - Upgrade from TiDB 5.2 versions to TiDB 5.3 versions.
 
+> **Warning:**
+>
+> - During an upgrade from earlier versions to v5.3, query errors may occur. To avoid this issue, you can set `set global tidb_allow_mpp = 0` and start the upgrade after completion of all queries in the MPP mode. After all components are upgraded to v5.3, you can set `set global tidb_allow_mpp = 1` to enable the MPP mode.
```

Comment:
```suggestion
> - During an upgrade of TiFlash from earlier versions to v5.3, query errors may occur. To avoid this issue, run `set global tidb_allow_mpp = 0` and start the upgrade after the completion of all queries in the MPP mode. After TiFlash is upgraded to v5.3, run `set global tidb_allow_mpp = 1` to enable the MPP mode.
```